src/user/sync.ts

```diff
@@ -284,6 +284,14 @@ async function Push(
   userProfile[storageKey] = storageURIs
 }
 
+const UploadLcps = async (): Promise<any> => {
+  const lcps = localStorage.getItem('extra_content.json')
+  Storage.put(`extra_content.json`, lcps, {
+    level: 'private',
+    contentType: 'text/plain',
+  }).catch(err => console.error(err))
+}
+
```

Comment:
Since `extra_content.json` is always empty after a bulk deletion, the deleted "content" previously was not pushed to the Cloud (because `lcps` equalled `''`, which is false-y).  Changed this to always push `extra_content.json` for a proper data overwrite.  Also extracted the upload into a separate function, in case we wanted to call it externally.